build.gradle.kts

```diff
@@ -48,6 +48,7 @@ repositories {
     mavenCentral()
     maven(url = "https://packages.confluent.io/maven/")
     maven(url = "https://jitpack.io")
+    maven(url = "https://build.shibboleth.net/maven/releases/")
```

Comment:
Må ha med denne for cxf 4.0 (siden den er avhengig av opensaml som ikke finnes på mavenCentral).